tests/unit/datarobot_drum/runtime_parameters/test_runtime_parameters.py

```diff
@@ -275,3 +277,140 @@ def test_setup_success(
 
                 actual_value = RuntimeParameters.get(param_name)
                 assert actual_value == expected_value
+
```

Comment:
`unit/datarobot_drum/runtime_parameters/test_runtime_parameters.py`  tests `datarobot_drum/runtime_parameters/runtime_parameters.py`  file.  these tests are in `drum.py` and so should be in the `test_drum.py` file.